dts/riscv/bouffalolab/bl60x.dtsi

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2021-2024 ATL Electronics
+ * Copyright (c) 2024 MASSDRIVER EI (massdriver.space)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <dt-bindings/pinctrl/bl60x-pinctrl.h>
+#include <dt-bindings/pinctrl/bflb-common-pinctrl.h>
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu0: cpu@0 {
+			clock-frequency = <0>;
+			compatible = "riscv,sifive-e24", "riscv";
+			device_type = "cpu";
+			hardware-exec-breakpoint-count = <4>;
+			reg = <0>;
+			riscv,isa = "rv32imafcb";
+			riscv,pmpregions = <4>;
+
+			ictrl: interrupt-controller {
+				#address-cells = <0>;
+				#interrupt-cells = <1>;
+				compatible = "riscv,cpu-intc";
+				interrupt-controller;
+			};
+		};
+	};
+
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "simple-bus";
+		ranges;
+
+		itcm: itcm@22010000 {
+			compatible = "zephyr,memory-region", "sifive,dtim0";
+			reg = <0x22010000 DT_SIZE_K(16)>;
+			zephyr,memory-region = "ITCM";
+		};
+		dtcm: dtcm@42014000 {
+			compatible = "zephyr,memory-region", "sifive,dtim0";
+			reg = <0x42014000 DT_SIZE_K(48)>;
+			zephyr,memory-region = "DTCM";
+		};
+
+		sram0: memory@42020000 {
+			compatible = "mmio-sram";
+			reg = <0x42020000 DT_SIZE_K(64)>;
+		};
+		sram1: memory@42030000 {
+			compatible = "mmio-sram";
+			reg = <0x42030000 DT_SIZE_K(112)>;
+		};
+
+		clic: clic@2000000 {
+			compatible = "sifive,clic";
+			reg = <0x2000000 0x10000>;
+			reg-names = "control";
+
+			#address-cells = <0>;
+			#interrupt-cells = <2>;
+			interrupt-controller;
+
+			interrupts-extended = <&ictrl 3 &ictrl 7 &ictrl 11 &ictrl 12>;
+			interrupt-names = "msip", /* Machine Software Interrupt */
+					  "mtip", /* Machine Timer interrupt    */
+					  "meip", /* Machine External Interrupt */
+					  "csip"; /* CLIC Software Interrupt    */
+			/*
+			 *	sifive,numintbits = <4>;
+			 *	sifive,numints = <64>;
+			 *	sifive,numlevels = <16>;
+			 */
```

Comment:
If we don't need their properties set, let's remove this comment.

Reply:
removed